automation/ip-address-script/create-ip.ps1

```diff
@@ -36,6 +40,14 @@ param (
     [string]
     $ResourceGroupName,
 
+    [Parameter(Mandatory=$False, HelpMessage='The service principal ID')]
+    [string]
+    $ServicePrincipalId,
+
+    [Parameter(Mandatory=$False, HelpMessage='The service principal password')]
+    [string]
+    $ServicePrincipalPassword,
+
```

Comment:
adding location as required parameter would avoid user input in case new resource group needs to be created

Reply:
also both service parameters are mandatory